client/js/_author/components/assessments/question_types/image_sequence/_image_sequence.jsx

```diff
@@ -0,0 +1,68 @@
+import React      from 'react';
+// import Settings from '../question_common/settings';
+import Feedback    from '../question_common/single_feedback';
+import ImageOrder  from './image_order';
+import types       from '../../../../../constants/question_types';
+
+export default class ImageSequence extends React.Component {
+  static propTypes = {
+    item: React.PropTypes.shape({
+      bankId: React.PropTypes.string,
+      answers: React.PropTypes.arrayOf(React.PropTypes.shape),
+      id: React.PropTypes.string,
+      type: React.PropTypes.string,
+      multipleAnswer: React.PropTypes.string,
+      question: React.PropTypes.shape({
+        choices: React.PropTypes.shape({}),
+      }),
+    }).isRequired,
+    updateItem: React.PropTypes.func,
+  };
+
+  constructor() {
+    super();
+    this.state = {
+    };
+  }
+
+  getFeedback() {
+    const { question, type } = this.props.item;
+
+    if (type !== types.multipleChoice) {
```

Comment:
The type will never be multiple choice, you should probably pull the logic out and just inline this stuff